ods_ci/tests/Resources/Page/ModelRegistry/ModelRegistry.resource

```diff
@@ -404,3 +411,37 @@ Post Upgrade Scenario Teardown
     Remove Deployment Files    ${MODELREGISTRY_BASE_FOLDER}/Python_Dependencies
     Disable Model Registry If Needed
     RHOSi Teardown
+
+Get Model Registry Operator Pod
+    [Documentation]    Returns the name of the operator pod for model registry
+    ${rc}    ${name} =    Run And Return Rc And Output
+    ...    oc get pod -l component.opendatahub.io/name=model-registry-operator -n ${APPLICATIONS_NAMESPACE} -o yaml | yq '.items[0].metadata.name'
+    Should Be Equal As Integers    ${rc}    0
+    RETURN    ${name}
+
+Get Model Registry Operator ${type} Image
+    [Documentation]    Returns the GRPC image defined in the operator, type should be either GRPC or REST (case 
+    ...    sensitive)
+    IF    "${type}"!="GRPC" and "${type}"!="REST"
```

Comment:
it should be an OR condition

maybe you need to lower case them in the check at least? the keyword calls use both upper and lowercase